### PR TITLE
planner: update the SPM doc

### PR DESCRIPTION
**Pull request by qw4990**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

TiDB v7.6.0 support universal bindings. 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
https://github.com/qw4990/docs-cn/pull/1

**Review by Oreoxmt — APPROVED** [review]

File: sql-plan-management.md [hunk 1]
```diff
@@ -471,6 +477,134 @@ SHOW binding_cache status;
...
+创建通用绑定的语法如下：
+
+```sql
+CREATE [GLOBAL | SESSION] UNIVERSAL BINDING [FOR BindableStmt] USING BindableStmt;
+```
+
+例如：
+
+```sql
+CREATE GLOBAL UNIVERSAL BINDING USING SELECT /*+ use_index(t, a) */ * FROM t;
```
Comment (qiancai) [inline sql-plan-management.md:493]:
```suggestion
CREATE GLOBAL UNIVERSAL BINDING USING SELECT /*+ use_index(t, a) */ * FROM t; -- 创建 GLOBAL 作用域的通用绑定
```

File: sql-plan-management.md [hunk 2]
```diff
@@ -471,6 +477,134 @@ SHOW binding_cache status;
...
+CREATE GLOBAL BINDING USING SELECT /*+ use_index(t, a) */ * FROM t;
```
Comment (qiancai) [inline sql-plan-management.md:494]:
```suggestion
CREATE GLOBAL BINDING USING SELECT /*+ use_index(t, a) */ * FROM t; -- 创建 GLOBAL 作用域的普通绑定
```

File: sql-plan-management.md [hunk 3]
```diff
@@ -471,6 +477,134 @@ SHOW binding_cache status;
...
+| select * from `test` . `t` | SELECT /*+ use_index(`t` `a`)*/ * FROM `test`.`t` | test       | enabled | 2023-12-18 22:46:25.224 | 2023-12-18 22:46:25.224 | utf8    | utf8_general_ci | manual |      | 8b193b00413fdb910d39073e0d494c96ebf24d1e30b131ecdd553883d0e29b42 |             |
+| select * from `t`          | SELECT /*+ use_index(`t` `a`)*/ * FROM `t`        |            | enabled | 2023-12-18 22:43:55.509 | 2023-12-18 22:43:55.509 | utf8    | utf8_general_ci | manual | u    | e5796985ccafe2f71126ed6c0ac939ffa015a8c0744a24b7aee6d587103fd2f7 |             |
++----------------------------+---------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------+------------------------------------------------------------------+-------------+
+```
+
+在 `SHOW GLOBAL BINDINGS` 的输出结果中，`Type` 字段为 `u` 表示该条绑定为通用绑定，如果为空，则为普通绑定。与普通绑定不同的是，通用绑定的 `Original_sql` 和 `Bind_sql` 中不包含数据库名。因此，这条通用绑定会对所有的 `select * from t` 查询生效，而不限于特定数据库。
```
Comment (qiancai) [inline sql-plan-management.md:509]:
```suggestion
在 `SHOW GLOBAL BINDINGS` 的输出结果中，`Type` 字段为 `u` 表示该条绑定为通用绑定，如果为空，则为普通绑定。与普通绑定不同的是，通用绑定的 `Original_sql` 和 `Bind_sql` 字段中不包含数据库名。因此，这条通用绑定会对集群中所有的 `select * from t` 查询生效，而不限于特定数据库。
```

File: sql-plan-management.md [hunk 4]
```diff
@@ -471,6 +477,134 @@ SHOW binding_cache status;
...
+
+通用绑定与普通绑定可以同时存在，匹配的优先级从高到低依次为：SESSION 级别的普通绑定 > SESSION 级别的通用绑定 > GLOBAL 级别的普通绑定 > GLOBAL 级别的通用绑定。
```
Comment (qiancai) [inline sql-plan-management.md:511]:
```suggestion
对于相同的查询，通用绑定与普通绑定可以同时存在，TiDB 匹配的优先级从高到低依次为：SESSION 级别的普通绑定 > SESSION 级别的通用绑定 > GLOBAL 级别的普通绑定 > GLOBAL 级别的通用绑定。
```

File: sql-plan-management.md [hunk 5]
```diff
@@ -471,6 +477,134 @@ SHOW binding_cache status;
...
++----------------------------------+------------------------------------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------+------------------------------------------------------------------+-------------+
+| select * from ( `t1` ) join `t1` | SELECT /*+ use_index(`t1` `a`) use_index(`t1` `a`)*/ * FROM (`t1`) JOIN `t1` |            | enabled | 2023-12-22 16:04:35.802 | 2023-12-22 16:04:35.802 | utf8    | utf8_general_ci | manual | u    | 79d994e5e2fbf14dcf48ef96c05049cc9800c28f03f9cdb9be326fb59f2adb51 |             |
++----------------------------------+------------------------------------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------+------------------------------------------------------------------+-------------+
+```
+
+由于数据库名被移除掉了，会出现两条重复的 `use_index(t1 a)` 的 Hint，导致 Hint 可能无法正确的和表名匹配起来。
```
Comment (qiancai) [inline sql-plan-management.md:606]:
```suggestion
由于数据库名被移除掉了，会出现两条重复的 `use_index(`t1` `a`)` 的 Hint，可能导致 Hint 无法和表名准确匹配。
```

File: sql-plan-management.md [hunk 6]
```diff
@@ -471,6 +477,134 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## 跨数据库绑定执行计划 (Universal Binding)
+
+跨数据库绑定执行计划，即通用绑定 (Universal Binding) 可以匹配模式相同的 SQL 语句，即使这些 SQL 运行在不同的数据库上。在使用通用绑定前，需要开启 [`tidb_opt_enable_universal_binding`](/system-variables.md#tidb_opt_enable_universal_binding-从-v760-版本开始引入) 系统变量。
```
Comment (qiancai) [inline sql-plan-management.md:482]:
```suggestion
跨数据库绑定执行计划，即通用绑定 (Universal Binding) 可以为模式相同的 SQL 语句绑定相同的执行计划，即使这些 SQL 运行在不同的数据库上。在使用通用绑定前，需要开启 [`tidb_opt_enable_universal_binding`](/system-variables.md#tidb_opt_enable_universal_binding-从-v760-版本开始引入) 系统变量。
```

File: sql-plan-management.md [hunk 7]
```diff
@@ -471,6 +477,134 @@ SHOW binding_cache status;
...
+| select * from `test` . `t` | SELECT /*+ use_index(`t` `a`)*/ * FROM `test`.`t` | test       | enabled | 2023-12-18 22:46:25.224 | 2023-12-18 22:46:25.224 | utf8    | utf8_general_ci | manual |      | 8b193b00413fdb910d39073e0d494c96ebf24d1e30b131ecdd553883d0e29b42 |             |
+| select * from `t`          | SELECT /*+ use_index(`t` `a`)*/ * FROM `t`        |            | enabled | 2023-12-18 22:43:55.509 | 2023-12-18 22:43:55.509 | utf8    | utf8_general_ci | manual | u    | e5796985ccafe2f71126ed6c0ac939ffa015a8c0744a24b7aee6d587103fd2f7 |             |
++----------------------------+---------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------+------------------------------------------------------------------+-------------+
+```
+
+在 `SHOW GLOBAL BINDINGS` 的输出结果中，`Type` 字段为 `u` 表示该条绑定为通用绑定，如果为空，则为普通绑定。与普通绑定不同的是，通用绑定的 `Original_sql` 和 `Bind_sql` 中不包含数据库名。因此，这条通用绑定会对所有的 `select * from t` 查询生效，而不限于特定数据库。
+
+通用绑定与普通绑定可以同时存在，匹配的优先级从高到低依次为：SESSION 级别的普通绑定 > SESSION 级别的通用绑定 > GLOBAL 级别的普通绑定 > GLOBAL 级别的通用绑定。
+
+除了创建方式不同，通用绑定的删除和状态设置与普通绑定相同。下面是一个完整的使用示例。
```
Comment (qiancai) [inline sql-plan-management.md:513]:
```suggestion
除了创建方式不同，通用绑定的删除和状态变更语句与普通绑定相同。下面是一个更加详细的使用示例。
```

Comment (qiancai) [reply sql-plan-management.md:513]: 也不算完整，因为没有包含绑定状态变更的示例

File: sql-plan-management.md [hunk 8]
```diff
@@ -471,6 +477,134 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## 跨数据库绑定执行计划 (Universal Binding)
+
+跨数据库绑定执行计划，即通用绑定 (Universal Binding) 可以为模式相同的 SQL 语句绑定相同的执行计划，即使这些 SQL 运行在不同的数据库上。TiDB 从 v7.6.0 开始引入通用绑定。在使用通用绑定前，需要开启 [`tidb_opt_enable_universal_binding`](/system-variables.md#tidb_opt_enable_universal_binding-从-v760-版本开始引入) 系统变量。
+
```
Comment (songrijie) [inline sql-plan-management.md:483]:
```suggestion

当用户将数据按照“数据库(scehma)”分开存储， 每个数据库拥有相同的对象定义，运行的业务逻辑也类似时，跨数据库执行计划绑定将会极大简化执行计划的固定工作。这样的建模一般有几个原因：

* 用户在 TiDB 上运行 SaaS 或 PaaS 类服务， 每个租户的数据存储在独立的数据库中，方便数据维护管理。 
* 用户在原单实例中做了分库操作，迁移到 TiDB 之后没有合库，而是沿用了原来的结构，把原先每个实例的数据按数据库分别存储。 

 "通用绑定" 能够有效缓解这类场景中一个常见问题。SaaS 或 PaaS 客户的数据及负载并不均衡，客户基于自身业务发展需要，数据量可能会剧烈变化。在这个场景下，由于统计信息无法保持及时，数据量由小及大的快速变化经常会引发 SQL 性能问题。在这种情况下，SaaS 服务商可以选择用 "通用绑定" 固定”大客户”已经验证过的执行计划，所有客户的执行计划都会被固定，避免“小用户”业务快速增长带来的潜在性能问题。

```

**Comment by qw4990** [comment]
Please hold on, we may update this feature's design this week.

File: sql-plan-management.md [hunk 9]
```diff
@@ -471,6 +476,122 @@ SHOW binding_cache status;
...
++----------------------------+---------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------------------------------------------------------------------+-------------+
+| select * from `test` . `t` | SELECT /*+ use_index(`t` `a`)*/ * FROM `test`.`t` | test       | enabled | 2023-12-29 14:19:01.332 | 2023-12-29 14:19:01.332 | utf8    | utf8_general_ci | manual | 8b193b00413fdb910d39073e0d494c96ebf24d1e30b131ecdd553883d0e29b42 |             |
+| select * from `*` . `t`    | SELECT /*+ use_index(`t` `a`)*/ * FROM `*`.`t`    |            | enabled | 2023-12-29 14:19:02.232 | 2023-12-29 14:19:02.232 | utf8    | utf8_general_ci | manual | 8b193b00413fdb910d39073e0d494c96ebf24d1e30b131ecdd553883d0e29b42 |             |
++----------------------------+---------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------------------------------------------------------------------+-------------+
+
```
Comment (Oreoxmt) [inline sql-plan-management.md:507]:
```suggestion
```

File: sql-plan-management.md [hunk 10]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## 跨数据库绑定执行计划 (Cross-DB Binding)
+
+TiDB 支持使用通配符 '*' 来表示数据库，实现跨数据库绑定。TiDB 从 v7.6.0 引入跨数据库绑定，在使用跨数据库绑定前，需要开启 [`tidb_opt_enable_fuzzy_binding`](/system-variables.md#tidb_opt_enable_fuzzy_binding-从-v760-版本开始引入) 系统变量
```
Comment (Oreoxmt) [inline sql-plan-management.md:481]:
```suggestion
TiDB 支持使用通配符 `*` 表示数据库，实现跨数据库绑定。该功能自 v7.6.0 开始引入。要使用跨数据库绑定前，首先需要开启 [`tidb_opt_enable_fuzzy_binding`](/system-variables.md#tidb_opt_enable_fuzzy_binding-从-v760-版本开始引入) 系统变量。
```

File: sql-plan-management.md [hunk 11]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+
+当用户将数据按照“数据库(scehma/db)”分开存储， 每个数据库拥有相同的对象定义，运行的业务逻辑也类似时，跨数据库执行计划绑定将会极大简化执行计划的固定工作。这样的建模一般有几个原因：
```
Comment (Oreoxmt) [inline sql-plan-management.md:483]:
```suggestion
当数据按数据库 (schema/db) 分类存储，同时各数据库具有相同的对象定义并且运行相似的业务逻辑时，跨数据库执行计划绑定能显著简化执行计划的固定过程。以下是一些常见的使用场景：
```

File: sql-plan-management.md [hunk 12]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+
+* 用户在 TiDB 上运行 SaaS 或 PaaS 类服务， 每个租户的数据存储在独立的数据库中，方便数据维护管理。 
+* 用户在原单实例中做了分库操作，迁移到 TiDB 之后没有合库，而是沿用了原来的结构，把原先每个实例的数据按数据库分别存储。
```
Comment (Oreoxmt) [inline sql-plan-management.md:486]:
```suggestion
* 用户在 TiDB 上运行 SaaS 或 PaaS 类服务，每个租户的数据存储于独立的数据库中，以便数据维护和管理。 
* 用户在单一实例中进行分库操作，并在迁移到 TiDB 后保留了原有的数据库结构，即将原实例中的数据按数据库分类存储。 
```

File: sql-plan-management.md [hunk 13]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+## 跨数据库绑定执行计划 (Cross-DB Binding)
+
+TiDB 支持使用通配符 '*' 来表示数据库，实现跨数据库绑定。TiDB 从 v7.6.0 引入跨数据库绑定，在使用跨数据库绑定前，需要开启 [`tidb_opt_enable_fuzzy_binding`](/system-variables.md#tidb_opt_enable_fuzzy_binding-从-v760-版本开始引入) 系统变量
+
+当用户将数据按照“数据库(scehma/db)”分开存储， 每个数据库拥有相同的对象定义，运行的业务逻辑也类似时，跨数据库执行计划绑定将会极大简化执行计划的固定工作。这样的建模一般有几个原因：
+
+* 用户在 TiDB 上运行 SaaS 或 PaaS 类服务， 每个租户的数据存储在独立的数据库中，方便数据维护管理。 
+* 用户在原单实例中做了分库操作，迁移到 TiDB 之后没有合库，而是沿用了原来的结构，把原先每个实例的数据按数据库分别存储。 
+
+ "跨数据库绑定" 能够有效缓解这类场景中一个常见问题。SaaS 或 PaaS 客户的数据及负载并不均衡，客户基于自身业务发展需要，数据量可能会剧烈变化。在这个场景下，由于统计信息无法保持及时，数据量由小及大的快速变化经常会引发 SQL 性能问题。在这种情况下，SaaS 服务商可以选择用 "跨数据库绑定" 固定”大客户”已经验证过的执行计划，所有客户的执行计划都会被固定，避免“小用户”业务快速增长带来的潜在性能问题。
```
Comment (Oreoxmt) [inline sql-plan-management.md:488]:
```suggestion
在这些场景中，跨数据库绑定能有效缓解由于用户数据和负载的不均衡及其快速变化所引发的 SQL 性能问题。SaaS 服务商可以通过跨数据库绑定，固定“大用户”已验证的执行计划，从而避免因“小用户”业务快速增长引起的潜在性能问题。
```

File: sql-plan-management.md [hunk 14]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+
+使用跨数据库绑定，只需要将数据库名用 '*' 表示，例如：
```
Comment (Oreoxmt) [inline sql-plan-management.md:490]:
```suggestion
使用跨数据库绑定，只需要将数据库名用 `*` 表示，例如：
```

File: sql-plan-management.md [hunk 15]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+| select * from `test` . `t` | SELECT /*+ use_index(`t` `a`)*/ * FROM `test`.`t` | test       | enabled | 2023-12-29 14:19:01.332 | 2023-12-29 14:19:01.332 | utf8    | utf8_general_ci | manual | 8b193b00413fdb910d39073e0d494c96ebf24d1e30b131ecdd553883d0e29b42 |             |
+| select * from `*` . `t`    | SELECT /*+ use_index(`t` `a`)*/ * FROM `*`.`t`    |            | enabled | 2023-12-29 14:19:02.232 | 2023-12-29 14:19:02.232 | utf8    | utf8_general_ci | manual | 8b193b00413fdb910d39073e0d494c96ebf24d1e30b131ecdd553883d0e29b42 |             |
++----------------------------+---------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------------------------------------------------------------------+-------------+
+```
+
+在 `SHOW GLOBAL BINDINGS` 的输出结果中，跨数据库绑定的 `Default_db` 为空，且 `Original_sql` 和 `Bind_sql` 字段中的数据库名通过 `*` 表示。这条绑定会对所用的 `select * from t` 查询生效，却不限定与特定数据库。
```
Comment (Oreoxmt) [inline sql-plan-management.md:509]:
```suggestion
在 `SHOW GLOBAL BINDINGS` 的输出结果中，跨数据库绑定的 `Default_db` 为空，且 `Original_sql` 和 `Bind_sql` 字段中的数据库名通过 `*` 表示。这条绑定会对所有 `select * from t` 查询生效，而不限于特定数据库。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by songrijie — APPROVED** [review]

File: sql-plan-management.md [hunk 16]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+TiDB 支持使用通配符 `*` 表示数据库，实现跨数据库绑定。该功能自 v7.6.0 开始引入。要使用跨数据库绑定，首先需要开启 [`tidb_opt_enable_fuzzy_binding`](/system-variables.md#tidb_opt_enable_fuzzy_binding-从-v760-版本开始引入) 系统变量。
+
+当数据按数据库 (schema/db) 分类存储，同时各数据库具有相同的对象定义并且运行相似的业务逻辑时，跨数据库执行计划绑定能显著简化执行计划的固定过程。以下是一些常见的使用场景：
+
+* 用户在 TiDB 上运行 SaaS 或 PaaS 类服务，每个租户的数据存储于独立的数据库中，以便数据维护和管理。 
+* 用户在单一实例中进行分库操作，并在迁移到 TiDB 后保留了原有的数据库结构，即将原实例中的数据按数据库分类存储。 
+
+在这些场景中，跨数据库绑定能有效缓解由于用户数据和负载的不均衡及其快速变化所引发的 SQL 性能问题。SaaS 服务商可以通过跨数据库绑定，固定大数据量用户已验证的执行计划，从而避免因小数据量用户业务快速增长引起的潜在性能问题。
+
+使用跨数据库绑定，只需要将数据库名用 `*` 表示，例如：
```
Comment (qiancai) [inline sql-plan-management.md:490]:
```suggestion
使用跨数据库绑定，只需要在创建绑定的 SQL 语句中将数据库名用 `*` 表示，例如：
```

File: sql-plan-management.md [hunk 17]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+| Original_sql               | Bind_sql                                          | Default_db | Status  | Create_time             | Update_time             | Charset | Collation       | Source | Sql_digest                                                       | Plan_digest |
++----------------------------+---------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------------------------------------------------------------------+-------------+
+| select * from `test` . `t` | SELECT /*+ use_index(`t` `a`)*/ * FROM `test`.`t` | test       | enabled | 2023-12-29 14:19:01.332 | 2023-12-29 14:19:01.332 | utf8    | utf8_general_ci | manual | 8b193b00413fdb910d39073e0d494c96ebf24d1e30b131ecdd553883d0e29b42 |             |
+| select * from `*` . `t`    | SELECT /*+ use_index(`t` `a`)*/ * FROM `*`.`t`    |            | enabled | 2023-12-29 14:19:02.232 | 2023-12-29 14:19:02.232 | utf8    | utf8_general_ci | manual | 8b193b00413fdb910d39073e0d494c96ebf24d1e30b131ecdd553883d0e29b42 |             |
```
Comment (qiancai) [inline sql-plan-management.md:505]: L493/L494 里跨数据库绑定在前面，L504/505 里跨数据库绑定在后面，需要顺序一致吗

File: sql-plan-management.md [hunk 18]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
++----------------------------+---------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------------------------------------------------------------------+-------------+
+```
+
+在 `SHOW GLOBAL BINDINGS` 的输出结果中，跨数据库绑定的 `Default_db` 为空，且 `Original_sql` 和 `Bind_sql` 字段中的数据库名通过 `*` 表示。这条绑定会对所有 `select * from t` 查询生效，而不限于特定数据库。
+
+对于相同的查询，跨数据绑定与普通绑定可以同时存在，TiDB 匹配的优先级从高到低依次为：SESSION 级别的普通绑定 > SESSION 级别的跨数据库绑定 > GLOBAL 级别的普通绑定 > GLOBAL 级别的跨数据库绑定。
+
+除了创建方式不同，跨数据库绑定的删除和状态变更语句与普通绑定相同。下面是一个详细的使用示例。
```
Comment (qiancai) [inline sql-plan-management.md:513]:
```suggestion
除了创建绑定的 SQL 语句不同，跨数据库绑定的删除和状态变更语句与普通绑定相同。下面是一个详细的使用示例。
```

File: sql-plan-management.md [hunk 19]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+    +----------------------------------+------------------------------------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------------------------------------------------------------------+-------------+
+    | select * from ( `*` . `t1` ) join `*` . `t2` | SELECT /*+ use_index(`t1` `a`) use_index(`t2` `a`)*/ * FROM (`*` . `t1`) JOIN `*` . `t2` |            | enabled | 2023-12-29 14:22:28.144 | 2023-12-29 14:22:28.144 | utf8    | utf8_general_ci | manual | ea8720583e80644b58877663eafb3579700e5f918a748be222c5b741a696daf4 |             |
+    +----------------------------------+------------------------------------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------------------------------------------------------------------+-------------+
+    ```
+
```
Comment (qiancai) [inline sql-plan-management.md:585]: 结果中的对齐看起来有些问题

image

File: sql-plan-management.md [hunk 20]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## 跨数据库绑定执行计划 (Cross-DB Binding)
+
+TiDB 支持使用通配符 `*` 表示数据库，实现跨数据库绑定。该功能自 v7.6.0 开始引入。要使用跨数据库绑定，首先需要开启 [`tidb_opt_enable_fuzzy_binding`](/system-variables.md#tidb_opt_enable_fuzzy_binding-从-v760-版本开始引入) 系统变量。
```
Comment (qiancai) [inline sql-plan-management.md:481]:
```suggestion
在创建绑定的 SQL 语句中，TiDB 支持使用通配符 `*` 表示数据库，实现跨数据库绑定。该功能自 v7.6.0 开始引入。要使用跨数据库绑定，首先需要开启 [`tidb_opt_enable_fuzzy_binding`](/system-variables.md#tidb_opt_enable_fuzzy_binding-从-v760-版本开始引入) 系统变量。
```

File: sql-plan-management.md [hunk 21]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+
+当数据按数据库 (schema/db) 分类存储，同时各数据库具有相同的对象定义并且运行相似的业务逻辑时，跨数据库执行计划绑定能显著简化执行计划的固定过程。以下是一些常见的使用场景：
+
+* 用户在 TiDB 上运行 SaaS 或 PaaS 类服务，每个租户的数据存储于独立的数据库中，以便数据维护和管理。 
+* 用户在单一实例中进行分库操作，并在迁移到 TiDB 后保留了原有的数据库结构，即将原实例中的数据按数据库分类存储。 
+
+在这些场景中，跨数据库绑定能有效缓解由于用户数据和负载的不均衡及其快速变化所引发的 SQL 性能问题。SaaS 服务商可以通过跨数据库绑定，固定大数据量用户已验证的执行计划，从而避免因小数据量用户业务快速增长引起的潜在性能问题。
```
Comment (qiancai) [inline sql-plan-management.md:488]:
```suggestion
在这些场景中，跨数据库绑定能有效缓解由于用户数据和负载的不均衡及其快速变化所引发的 SQL 性能问题。SaaS 服务商可以通过跨数据库绑定，固定大数据量用户业务已验证的执行计划，从而避免因小数据量用户业务快速增长引起的潜在性能问题。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: sql-plan-management.md [hunk 22]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+* 用户在 TiDB 上运行 SaaS 或 PaaS 类服务，每个租户的数据存储于独立的数据库中，以便数据维护和管理。 
+* 用户在单一实例中进行分库操作，并在迁移到 TiDB 后保留了原有的数据库结构，即将原实例中的数据按数据库分类存储。 
+
+在这些场景中，跨数据库绑定能有效缓解由于用户数据和负载的不均衡及其快速变化所引发的 SQL 性能问题。SaaS 服务商可以通过跨数据库绑定，固定大数据量用户业务已验证的执行计划，从而避免因小数据量用户业务快速增长引起的潜在性能问题。
+
+使用跨数据库绑定，只需要在创建绑定的 SQL 语句中将数据库名用 `*` 表示，例如：
+
+```sql
+CREATE GLOBAL BINDING USING SELECT /*+ use_index(t, a) */ * FROM *.t; -- 创建 GLOBAL 作用域的跨数据库绑定
+CREATE GLOBAL BINDING USING SELECT /*+ use_index(t, a) */ * FROM t; -- 创建 GLOBAL 作用域的普通绑定
```
Comment (qw4990) [inline sql-plan-management.md:494]:
```suggestion
CREATE GLOBAL BINDING USING SELECT /*+ use_index(t, a) */ * FROM t; -- 创建 GLOBAL 作用域的普通绑定
CREATE GLOBAL BINDING USING SELECT /*+ use_index(t, a) */ * FROM *.t; -- 创建 GLOBAL 作用域的跨数据库绑定
```

File: sql-plan-management.md (hunk 17 above)
Comment (qw4990) [reply sql-plan-management.md:505]: 我把 L493/L494 的部分调整了

File: sql-plan-management.md [hunk 23]
```diff
@@ -471,6 +476,121 @@ SHOW binding_cache status;
...
+    +----------------------------------+------------------------------------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------------------------------------------------------------------+-------------+
+    | select * from ( `*` . `t1` ) join `*` . `t2` | SELECT /*+ use_index(`t1` `a`) use_index(`t2` `a`)*/ * FROM (`*` . `t1`) JOIN `*` . `t2` |            | enabled | 2023-12-29 14:22:28.144 | 2023-12-29 14:22:28.144 | utf8    | utf8_general_ci | manual | ea8720583e80644b58877663eafb3579700e5f918a748be222c5b741a696daf4 |             |
+    +----------------------------------+------------------------------------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------------------------------------------------------------------+-------------+
```
Comment (Oreoxmt) [inline sql-plan-management.md:583]:
```suggestion
    +----------------------------------------------+------------------------------------------------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+--------------------+--------+------------------------------------------------------------------+-------------+
    | Original_sql                                 | Bind_sql                                                                                 | Default_db | Status  | Create_time             | Update_time             | Charset | Collation          | Source | Sql_digest                                                       | Plan_digest |
    +----------------------------------------------+------------------------------------------------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+--------------------+--------+------------------------------------------------------------------+-------------+
    | select * from ( `*` . `t1` ) join `*` . `t2` | SELECT /*+ use_index(`t1` `a`) use_index(`t2` `a`)*/ * FROM (`*` . `t1`) JOIN `*` . `t2` |            | enabled | 2023-12-29 14:22:28.144 | 2023-12-29 14:22:28.144 | utf8    | utf8_general_ci    | manual | ea8720583e80644b58877663eafb3579700e5f918a748be222c5b741a696daf4 |             |
    +----------------------------------------------+------------------------------------------------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+--------------------+--------+------------------------------------------------------------------+-------------+
```

**Pull request merged**